### ISSUE TITLE
Rotations empty on KiCAD 8.0

### ISSUE THREAD

**Issue by netkruzer** [Bouni/kicad-jlcpcb-tools#456]
Rotations are empty:
<img width="785" alt="image" src="https://github.com/Bouni/kicad-jlcpcb-tools/assets/7841302/d0672377-9a32-40fa-bb0a-adbdc26a3609">

**Describe the bug**
All rotations are empty. I used this plugin with KiCAD 7.x and it exported the rotations great. In 8.0 it no longer exports or shows the fixed rotations. Importing into JLCPCB's assembly menu, all rotations are off. 

<img width="312" alt="image" src="https://github.com/Bouni/kicad-jlcpcb-tools/assets/7841302/eacee4a3-2766-4620-894b-fb4c292d44db">
These same rotation mappings worked for me in KiCAD 7.x

JLCPCB Tools version 2024.04.03

**Expected behavior**
Should have rotations

# KiCad Version

```
Application: KiCad PCB Editor x64 on x64

Version: 8.0.1, release build

Libraries:
	wxWidgets 3.2.4
	FreeType 2.12.1
	HarfBuzz 8.3.0
	FontConfig 2.14.2
	libcurl/8.5.0-DEV Schannel zlib/1.3

Platform: Windows 10 (build 19045), 64-bit edition, 64 bit, Little endian, wxMSW

Build Info:
	Date: Mar 15 2024 01:52:47
	wxWidgets: 3.2.4 (wchar_t,wx containers)
	Boost: 1.83.0
	OCC: 7.7.1
	Curl: 8.5.0-DEV
	ngspice: 42
	Compiler: Visual C++ 1936 without C++ ABI

Build settings:

```

**Comment by lk-davidegironi** [comment]
Same problem to me on KiCad 7.0.11 on Win 10, JLCPCB Tools version 2024.04.03

![Capture](https://github.com/Bouni/kicad-jlcpcb-tools/assets/90449776/304b04d3-0813-4334-b1fd-8ce930bb20cd)

**Comment by Bouni** [comment]
Fixed in #445
